com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/ResolveMainClassHandler.java

```diff
@@ -40,19 +44,35 @@
 
 public class ResolveMainClassHandler {
     private static final Logger logger = Logger.getLogger(Configuration.LOGGER_NAME);
+    private static final String CLASSNAME_REGX = "([$\\w]+\\.)*[$\\w]+";
```

Comment:
I think a better way is to validate the main Class in js side.

Reply:
I don't want to separate the validation logic to two parts, that's hard for reading. JS side is responsible for UI related logic, and JAVA side is for computing the result.